src/Microsoft.Fx.Portability.Offline/OfflineApiPortService.cs

```diff
@@ -57,6 +58,13 @@ public Task<ServiceResponse<AnalyzeResponse>> SendAnalysisAsync(AnalyzeRequest a
         public Task<ServiceResponse<IEnumerable<ReportingResultWithFormat>>> SendAnalysisAsync(AnalyzeRequest a, IEnumerable<string> formats)
         {
             var response = _requestAnalyzer.AnalyzeRequest(a, Guid.NewGuid().ToString());
+
+            if(formats == null || !formats.Any())
+            {
+                var defaultFormat = GetDefaultResultFormatAsync().Result.Response;
```

Comment:
We shouldn't mix synchronous blocking calls (ie. `.Wait()` or `.Result`) with async methods.  I'd make this into an async method like:
```csharp
public async Task<ServiceResponse<IEnumerable<ReportingResultWithFormat>>> SendAnalysisAsync(AnalyzeRequest a, IEnumerable<string> formats)
{
...
                var serviceResponse = await GetDefaultResultFormatAsync();
                var defaultFormat = serviceResponse.Response;
...
}
```